lib/binary_to_decimal.rb

```diff
@@ -4,6 +4,16 @@
 # The least significant bit is at index 7.
 # Calculate  and return the decimal value for this binary number using
 # the algorithm you devised in class.
-def binary_to_decimal(binary_array)
-  raise NotImplementedError
+# def binary_to_decimal(binary_array)
+#  raise NotImplementedError
+# end
+
+def binary_to_decimal(bi_num)
```

Comment:
This works, but can you do it without the enumerable method `reverse`.

Reply:
Yep! Also added .idea to my global .gitignore. Thanks!